app/frontend/src/components/views/project-view/components/TaskPopup.tsx

```diff
@@ -18,22 +18,21 @@ const TaskPopup: React.FC = () => {
   const params = useParams();
   const projects = useStore($projects);
 
-  const id = params.task_id;
+  const id = params.task_id as string;
```

Comment:
If `task_id` is number, better use `.toString()`, not cast

```suggestion
  const id = params.task_id as string;
```